lego/apps/surveys/templates/surveys/email/survey.html

```diff
@@ -0,0 +1,40 @@
+{% extends "email/base.html" %}
+
+{% block alert %}
+
+    <tr>
+        <td class="alert alert-warning">
+            Du har en ny undersøkelse å svare på
+        </td>
+    </tr>
+
+{% endblock %}
+
+{% block content %}
+
+    <tr>
+        <td class="content-block">
+            Hei, {{ name }}!
+        </td>
+    </tr>
+
+    <tr>
+        <td class="content-block">
+            Du har en ny undersøkelse å svare på for arrangement {{ event }}.
+        </td>
+    </tr>
+
+    <tr>
+        <td class="content-block">
+            <i>Å ikke svare på undersøkelsen innen fristen vil medføre en prikk.</i>
```

Comment:
don't have to change this now maybe, but are we removing this when users can't join new events if they've got pending surveys?

Reply:
Yup. I think it would be a good idea to just implement this functionality straight away, shouldn't be too complicated :)

Reply:
Just rejecting the request to register backend would indeed not be too hard. However, we would also have to return something sensible to the front-end (list of unanswered surveys? Full objects or just ids?), then reduce that properly into the redux state (which would normally expect a Registration object), and finally display a proper error message to the end user. None of that is necessarily difficult, but it would take time to implement. Do we want to delay surveys launch for this functionality, if it comes to that?

Ofc feel free to just do it and submit a pr 🙃